app/modules/users/resources.py

```diff
@@ -106,7 +105,7 @@ def post(self, args):
             )
 
         if 'password' not in args:
-            args['password'] = _generate_salt(128)
+            abort(code=HTTPStatus.BAD_REQUEST, message='Must provide a password')
```

Comment:
I like this approach better.  We need to add the appropriate wrapper for `@api.response(code=HTTPStatus.BAD_REQUEST)` so that it will be documented appropriately